### PR TITLE
Modernize README and fix inaccuracies

### PR DESCRIPTION
**Pull request by kelsonic**
### Summary

We added a version status under the title and moved the build status to be beside it. We created Model, View, and Controller headers and changed the paragraphs from MCV order to MVC. We replaced the inaccurate "Yay! You're on Rails!" statement in step four of the Getting Started header with the Welcome to Rails page as an image. Removed step 5 in favor of giving it the Documentation, Guides, and Tutorials header.
### Other Information

We also added the Welcome to Rails page image to "/guides/assets/images/welcome_page.png". Finally, if you need any changes to our PR, please let us know and we will be happy to implement them.

Both @shaun-sweet and I (@kgadams93) worked on this together.

### CODE REVIEW

**Comment by maclover7** [comment]
I'd say this is a Rails Core / @dhh decision since it's such a visible part of the rails/rails repository 😬

**Comment by rafaelfranca** [comment]
Thank you for the pull request.

> We added a version status under the title and moved the build status to be beside it.

We don't want to add this kind of badges in the README. It is missing by design we chose to not put them. It doesn't say anything to our users. For example, the build should be always green, so the travis badge doesn't say anything. Besides that it is usually only useful for contributors, this is why the badge is in the CONTRIBUTING file. I have dificultty to understand how the version badge is useful too. Rails have a lot of different ways to tell users what is the latest version. In our web page the latest version is 5.0.0.beta3 (which is the correct one). This badge is pointing to 4.2.6. Also how users are supposed to use this information? If they want to know that the version they are using in out to date they have better ways to do it without entering the Rails repository like the `bundle outdated` command.

Also this change changes the 80 column limit we had to a longs lines and we prefer to keep the lines short.

The text was changed from a natural flow to an artificial set of topics. I really prefer the previous writing.

The welcoming screenshot is outdated. You are changing the master branch and in Rails 5 the welcome page is different of that one.

In my opinion, the only change we should apply is the inaccurate "Yay! You're on Rails!" statement. Could you change your PR for that?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 
-Rails is a web-application framework that includes everything needed to
-create database-backed web applications according to the
-[Model-View-Controller (MVC)](http://en.wikipedia.org/wiki/Model-view-controller)
-pattern.
+[![Gem Version](https://badge.fury.io/rb/rails.svg)](https://badge.fury.io/rb/rails) [![Build Status](https://travis-ci.org/rails/rails.svg?branch=master)](https://travis-ci.org/rails/rails)
 
-Understanding the MVC pattern is key to understanding Rails. MVC divides your
-application into three layers, each with a specific responsibility.
+A web framework that includes everything needed to create database-backed web applications using the [Model-View-Controller (MVC)](http://en.wikipedia.org/wiki/Model-view-controller) pattern. Understanding the MVC pattern is key to understanding Rails. MVC divides your application into three layers, each with a specific responsibility.
```
Comment (ianks) [inline README.md]: Can we shorten this line?

**Comment by ianks** [comment]
.DS_Store should be removed

File: README.md [hunk 2]
```diff
@@ -64,14 +66,17 @@ and may also be used independently outside Rails.
...
 4. Using a browser, go to `http://localhost:3000` and you'll see:
-"Yay! You’re on Rails!"
 
-5. Follow the guidelines to start developing your application. You may find
-   the following resources handy:
-    * [Getting Started with Rails](http://guides.rubyonrails.org/getting_started.html)
-    * [Ruby on Rails Guides](http://guides.rubyonrails.org)
-    * [The API Documentation](http://api.rubyonrails.org)
-    * [Ruby on Rails Tutorial](http://www.railstutorial.org/book)
+![Welcome to Rails Page](guides/assets/images/welcome_page.png "Welcome to Rails")
```
Comment (tvanderpol) [inline README.md]: This is out of date for master branch - the rails 5 welcome page has been changed

**Pull request closed without merging**